CHANGES.md

```diff
@@ -11,6 +11,7 @@ Features
 * [#1337](https://github.com/java-native-access/jna/pull/1337): Add `REG_NOTIFY_THREAD_AGNOSTIC` to `c.s.j.p.win32.WinNet` and update `REG_LEGAL_CHANGE_FILTER` - [@Dani-Hub](https://github.com/Dani-Hub).
 * [#1338](https://github.com/java-native-access/jna/pull/1338): Add `RegNotifyChangeKeyValue` to `c.s.j.p.win32.Advapi32` - [@Dani-Hub](https://github.com/Dani-Hub).
 * [#1340](https://github.com/java-native-access/jna/issues/1340): Added `CM_Get_DevNode_Registry_Property` to `c.s.j.p.win32.Cfgmgr32` and corresponding util in `c.s.j.p.win32.Cfgmgr32Util` - [@dbwiddis](https://github.com/dbwiddis).
+* [#1352](https://github.com/java-native-access/jna/pull/1352): Add `BringWindowToTop` to `c.s.j.p.win32.User32` - [@kahgoh](https://github.com/kahgoh)
```

Comment:
This is missing a period :)

Reply:
Not the only one to do so.  At least two entries for 5.7.0 lack ending full stops.  There's also a mix of "Issues" and "pull" in the URL that should probably be standardized.

At least it's not a missing semicolon in the code....

Reply:
LOL, maybe consider https://github.com/dblock/danger-changelog

Reply:
Damn! I wouldn't mind adding the missing full stop in the changelog to get it more consistent. Would another pull request be preferred?

Reply:
Please don't submit a PR for a single period.   If you want to go through the whole changelog and standardize things (all the periods and other fixes), it would probably be accepted.   Some thoughts:
 - Standard format
 - Fix urls from `issue` to `pull` when appropriate
 - Spell check

No need to be perfect, but if you're going to do a PR make at at least be substantive.

Reply:
Sorry I pushed https://github.com/java-native-access/jna/commit/80fa84ddbca4ecbd7bf44c3529720184f2688743 - yesterday evening my irony detector was giving strange signals and to me reading through this took longer, than to fix it, so I did that.

My take on this: Don't fix fix, that are not broken. For example the question could be asked: "What is the standard format?". This this the informal formatting currently established or is it some formal language like XML or JSON with an attached schema to be verifiable? For `issues` vs. `pulls` - I referenced issues if they were present (as the motivation for the change) and if not the PR.

Reply:
Thanks @matthiasblaesing! Personally, I have a bias towards consistent format because 1) my OCD 2) if we can't insist on the highest standards for a period, we are at risk to slip further on variable naming, documentation, unit tests, bugs ...